packages/react-scripts/template/README.md

```diff
@@ -301,6 +302,47 @@ becomes this:
 
 There is currently no support for preprocessors such as Less, or for sharing variables across CSS files.
 
+## Adding CSS Preprocessor (Sass, Less etc.)
+
+CSS preprocessors have become a vital part of build processes. Using a preprocesssor of your choice in a project bootstrapped using create-react-app, is fairly straightforward to setup, even without having to eject.
+
+First, install preprocessor of your choice. Sass seems the most popular weapon of choice at the moment, so we'll use it as an example.
+
+```
+npm install node-sass --save-dev
+```
+
+Then in `package.json` just add the following lines to `scripts`, replacing file paths accordingly.
+```
+  ...
+  "scripts": {
+    ...
+    "build-css": "node-sass src/sass/base.scss src/index.css",
+    "watch-css": "npm run build-css && node-sass src/sass/base.scss src/index.css -w",
+    ...
+  }
+  ...
+```
+
+> Using a different preprocessor should be just a matter of replacing `build-css` and `watch-css` scripts to something that matches the preprocessor you're using.
+
+Add these scripts to the main scripts, by pasting `npm run watch-css &` to `start` script and `npm run build-css &&` to `build`.
+
+```
+ ...
+  "scripts": {
+    "start": "npm run watch-css & react-scripts start",
+    "build": "npm run build-css && react-scripts build",
```

Comment:
` & ` won't work as intended on windows (it will wait for the watch process to exit before continuing). One possible solution is ` | `. See http://stackoverflow.com/questions/30950032/how-can-i-run-multiple-npm-scripts-in-parallel